tools/vcpkg/sgct/vcpkg.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "sgct",
```

Comment:
Should these not be added to the official vcpkg repo instead?

Reply:
Sure, ask the sgct developers to add it.